PPO/Networks.py

```diff
@@ -0,0 +1,49 @@
+import torch.nn as nn
+import numpy as np
+import torch
+from Config import Config
+import pdb
+
+# TODO:
+# - Layer initialisation
```

Comment:
I suppose you want to make this completely configurable correct?

Reply:
I'm actually not too sure, we can have the different networks with different initialisations, I don't really see a scenario where we wouldn't want to have it though WDYT?